### ISSUE TITLE
JetTabs won't load correct template if translated using WPML

### ISSUE THREAD

**Issue by mbaierl** [CrocoBlock/suggestions#4308]
**Describe the bug**
If JetTabs is using templates it will always load the default template, even if they have been translated to another language using WPML.

**To Reproduce**
1) Use JetTabs with templates
2) Translate one of these content templates using WPML
3) Switch to the secondary language
4) The template in the original language is loaded.

**Expected behavior**
The translated template is loaded and shown.

**Solution**
In the file `wp-content/plugins/jet-tabs/includes/rest-api/endpoints/elementor-template.php` in the function `public function callback( $request ) { ...` add the following code:

```
$template_id = apply_filters('wpml_object_id', $template_id, 'elementor_library', true);
```

This will ensure the translated template is loaded, if there is any.

Please fix this upstream, thank you!

**Comment by Floydtm** [comment]
We've checked your request and it looks like everything is working. If you have a problem with this, please contact our support team. Thanks
[bandicam 2021-11-15 11-12-24-465.zip](https://github.com/Crocoblock/suggestions/files/7537366/bandicam.2021-11-15.11-12-24-465.zip)

**Comment by mbaierl** [comment]
Hi @Floydtm - did you also try with Ajax loading on?

**Comment by Floydtm** [comment]
@mbaierl, hi! No, I have not enabled this option. But you didn’t say anything about it either. I will reopen this request. The developer will try to fix it. Thank you!

**Comment by mbaierl** [comment]
> @mbaierl, hi! No, I have not enabled this option. But you didn’t say anything about it either. I will reopen this request. The developer will try to fix it. Thank you!

Sorry, my fault!

**Comment by Floydtm** [comment]
Fix added to version 2.1.15
